### PR TITLE
Add new platform for Delta WB2448.

### PR DESCRIPTION
**Pull request by vincent0083**
Add new platform for Delta WB2448.

### CODE REVIEW

File: packages/platforms/delta/vendor-config/PKG.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+!include $ONL_TEMPLATES/platform-config-vendor.yml VENDOR=delta Vendor=DELTA
```
Comment (jnealtowns) [inline packages/platforms/delta/vendor-config/PKG.yml]: @vincent0083 Vendor=Delta

File: packages/platforms/delta/vendor-config/src/python/delta/__init__.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python
+
+from onl.platform.base import *
+
+class OnlPlatformDELTA(OnlPlatformBase):
```
Comment (jnealtowns) [inline packages/platforms/delta/vendor-config/src/python/delta/__init__.py]: OnlPlatformDelta(OnlPlatformBase).

File: packages/platforms/delta/vendor-config/src/python/delta/__init__.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    MANUFACTURER='DELTA'
```
Comment (jnealtowns) [inline packages/platforms/delta/vendor-config/src/python/delta/__init__.py]: 'Delta'

File: packages/platforms/delta/x86-64/x86-64-delta-wb2448/onlp/builds/src/module/auto/x86_64_delta_wb2448.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
+###############################################################################
+#
+# x86_64_delta_wb2448 Autogeneration Definitions.
+#
+###############################################################################
+
+cdefs: &cdefs
+- x86_64_delta_wb2448_CONFIG_INCLUDE_LOGGING:
```
Comment (jnealtowns) [inline packages/platforms/delta/x86-64/x86-64-delta-wb2448/onlp/builds/src/module/auto/x86_64_delta_wb2448.yml]: Config defines are by convention all uppercase. 

Please s/x86_64_delta_wb2448_CONFIG/X86_64_DELTA_WB2448_CONFIG/g

File: packages/platforms/delta/x86-64/x86-64-delta-wb2448/onlp/builds/src/module/src/sysi.c [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+onlp_sysi_platform_info_free(onlp_platform_info_t* pi)
+{
+    aim_free(pi->cpld_versions);
+}
+
+void
+onlp_sysi_onie_data_free(uint8_t* data)
+{
+    /*
+    * We returned a static array in onlp_sysi_onie_data_get()
```
Comment (jnealtowns) [inline packages/platforms/delta/x86-64/x86-64-delta-wb2448/onlp/builds/src/module/src/sysi.c]: This is not true. The data must be deallocated.

File: packages/platforms/delta/x86-64/x86-64-delta-wb2448/onlp/builds/src/module/src/thermali.c [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+        case THERMAL_3_ON_MAIN_BROAD:
+            rv = ifnOS_LINUX_BmcI2CGet(THERMAL_34_I2C_BUS, THERMAL_3_ADDR, THERMAL_REGISTER, THERMAL_ADDR_LEN, &u4Data, THERMAL_DATA_LEN);
+            break;
+            
+        case THERMAL_4_ON_MAIN_BROAD:
+            rv = ifnOS_LINUX_BmcI2CGet(THERMAL_34_I2C_BUS, THERMAL_4_ADDR, THERMAL_REGISTER, THERMAL_ADDR_LEN, &u4Data, THERMAL_DATA_LEN);
+            break;		
+            
+        default:
+            printf("Invalid Thermal ID!!\n");
```
Comment (jnealtowns) [inline packages/platforms/delta/x86-64/x86-64-delta-wb2448/onlp/builds/src/module/src/thermali.c]: printf's are not allowed. Please use AIM_LOG_ERROR()

File: packages/platforms/delta/x86-64/x86-64-delta-wb2448/platform-config/r0/src/python/x86_64_delta_wb2448_r0/__init__.py [hunk 7]
```diff
@@ -0,0 +1,16 @@
+from onl.platform.base import *
+from onl.platform.delta import *
+
+class OnlPlatform_x86_64_delta_wb2448_r0(OnlPlatformDELTA,
```
Comment (jnealtowns) [inline packages/platforms/delta/x86-64/x86-64-delta-wb2448/platform-config/r0/src/python/x86_64_delta_wb2448_r0/__init__.py]: OnlPlatformDelta

**Comment by jnealtowns** [comment]
@sonoble  Can you run a regression on the X86 systems with this change merged to make sure the configuration changes don't break anything? This will gate inclusion in master.

**Comment by sonoble** [comment]
No issues found.

**Pull request merged**